tests/core/gui/test_main_window.py

```diff
@@ -0,0 +1,194 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from PyQt5 import QtCore, QtWidgets
+
+from parsec.core.gui.lang import translate
+from parsec.core.types import BackendOrganizationFileLinkAddr
+
+
+@pytest.fixture
+async def logged_gui_with_files(
+    aqtbot, running_backend, backend, autoclose_dialog, logged_gui, bob, monkeypatch
+):
+    w_w = await logged_gui.test_switch_to_workspaces_widget()
+
+    assert logged_gui.tab_center.count() == 1
+
+    monkeypatch.setattr(
+        "parsec.core.gui.workspaces_widget.get_text_input", lambda *args, **kwargs: ("w1")
+    )
+    monkeypatch.setattr(
+        "parsec.core.gui.files_widget.get_text_input", lambda *args, **kwargs: ("dir1")
+    )
+
+    await aqtbot.mouse_click(w_w.button_add_workspace, QtCore.Qt.LeftButton)
+
+    def workspace_button_ready():
```

Comment:
nit-picking: start the private function name with an underscore ?

Reply:
fixed, thanks